python/doc/theory/data_analysis/quantile_estimation_wilks.rst

```diff
@@ -1,197 +1,90 @@
-| 717         | 691                                      | 682                                  |
-+-------------+------------------------------------------+--------------------------------------+
-| 740         | 713                                      | 704                                  |
-+-------------+------------------------------------------+--------------------------------------+
-| 763         | 735                                      | 725                                  |
-+-------------+------------------------------------------+--------------------------------------+
-| 786         | 757                                      | 747                                  |
-+-------------+------------------------------------------+--------------------------------------+
-| 809         | 779                                      | 769                                  |
-+-------------+------------------------------------------+--------------------------------------+
-| 832         | 801                                      | 791                                  |
-+-------------+------------------------------------------+--------------------------------------+
-| 855         | 823                                      | 813                                  |
-+-------------+------------------------------------------+--------------------------------------+
-| 877         | 844                                      | 834                                  |
-+-------------+------------------------------------------+--------------------------------------+
-| 900         | 866                                      | 856                                  |
-+-------------+------------------------------------------+--------------------------------------+
-| 923         | 888                                      | 877                                  |
-+-------------+------------------------------------------+--------------------------------------+
-| 945         | 909                                      | 898                                  |
-+-------------+------------------------------------------+--------------------------------------+
-| 968         | 931                                      | 920                                  |
-+-------------+------------------------------------------+--------------------------------------+
-| 991         | 953                                      | 942                                  |
-+-------------+------------------------------------------+--------------------------------------+
-
-:math:`\widehat{q}_{Y^i}(\alpha)` is often called the “empirical
-:math:`\alpha`-quantile” for the variable :math:`{Y^i}`.
+    k_{sol} & = \min \{ k \in \llbracket 1, n \rrbracket \, | \, F_{\sampleSize, \alpha}(k-1)\geq \beta \}\\
+            & = 1 +  \min \{ k \in \llbracket 1, n \rrbracket \, | \, F_{\sampleSize, \alpha}(k)\geq \beta \}
```

Comment:
it doesnt like the llbracket/rrbracket, goes fine with [ ]

Reply:
But I introduced the package stmaryd in the file math_notations.sty! I retry because I guess I had not pushed the new version of the math_notations.sty file....